oauth2client/django_util/storage.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2015 Google Inc.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from oauth2client.client import OAuth2Credentials, Storage
+
+
+def get_storage(request):
+    # TODO: Make this pluggable with different storage providers
```

Comment:
I see you've already created an issue - please format this as "TODO(issue 319): <text>".